### PR TITLE
netmasks autodiscovery&api key authentication fix

### PR DESCRIPTION
**Pull request by sa-cloud**

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -81,8 +81,9 @@ storage:
 exporter:
   enabled: false
   image:
-    repository: docker.io/bluesecure/skydive-exporter
-    tag: 1
+    #skydive/skydive-flow-exporter:master
```
Comment (hunchback) [inline values.yaml]: please remove comment

**Comment by hunchback** [comment]
Please work with @lebauce to add CI tests ... for now please paste results of `helm install` and `helm remove` as proof that changes don't break the code

**Comment by sa-cloud** [comment]
Demonstrating hem install:
```
helm install --name skydive --namespace default --set exporter.enabled=true .
NAME:   skydive
LAST DEPLOYED: Sun Feb  9 09:26:56 2020
NAMESPACE: default
STATUS: DEPLOYED

RESOURCES:
==> v1/Secret
NAME                     TYPE    DATA  AGE
skydive-exporter-secret  Opaque  3     3s

==> v1/ConfigMap
NAME                               DATA  AGE
skydive-exporter-s3-configuration  5     3s

==> v1/ServiceAccount
NAME                     SECRETS  AGE
skydive-service-account  1        3s

==> v1/ClusterRoleBinding
NAME                  AGE
skydive-default-view  3s

==> v1/Service
NAME                              TYPE       CLUSTER-IP      EXTERNAL-IP  PORT(S)             AGE
skydive-skydive-internal-service  ClusterIP  172.21.194.126  <none>       12379/TCP,9200/TCP  3s
skydive-skydive-service           NodePort   172.21.191.217  <none>       8082:30916/TCP      2s

==> v1beta1/DaemonSet
NAME                   DESIRED  CURRENT  READY  UP-TO-DATE  AVAILABLE  NODE SELECTOR  AGE
skydive-skydive-agent  3        3        0      3           0          <none>         2s

==> v1beta1/Deployment
NAME                      DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
skydive-skydive-analyzer  1        1        1           0          2s

==> v1/Pod(related)
NAME                                       READY  STATUS             RESTARTS  AGE
skydive-skydive-agent-8kwsr                0/1    Running            0         2s
skydive-skydive-agent-hqpbw                0/1    Running            0         2s
skydive-skydive-agent-nlh7l                0/1    Running            0         2s
skydive-skydive-analyzer-6cc575468f-sjg77  0/3    ContainerCreating  0         2s


NOTES:
To access skydive GUI execute:

(Bear with us couple of seconds until Skydive starts ...)
export UI_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].nodePort}" services skydive-skydive-service)
export UI_IP=$(kubectl get nodes --namespace default -o jsonpath="{.items[0].status.addresses[0].address}")
echo "skydive end-point: http://$UI_IP:$UI_PORT"
```

**Comment by sa-cloud** [comment]
Proof that the exporter and minio containers are up and running within the analyzer pod:
```
kubectl1 describe pod skydive-skydive-analyzer-6cc575468f-sjg77
Name:               skydive-skydive-analyzer-6cc575468f-sjg77
Namespace:          default
Priority:           0
PriorityClassName:  <none>
Node:               10.74.144.77/10.74.144.77
Start Time:         Sun, 09 Feb 2020 09:26:57 +0000
Labels:             app=skydive-skydive
                    chart=skydive
                    heritage=Tiller
                    pod-template-hash=6cc575468f
                    release=skydive
                    tier=analyzer
Annotations:        kubernetes.io/psp: ibm-privileged-psp
                    productID: 8e6bdbcba44f46939c3d1c54447386b2
                    productName: skydive
                    productVersion: 1.1.2
Status:             Running
IP:                 10.74.144.77
Controlled By:      ReplicaSet/skydive-skydive-analyzer-6cc575468f
Containers:
  skydive-analyzer:
    Container ID:  containerd://ac39280fefe7f6627a939e766f96b4962a289652256cca36f4b6752ebe4d49f7
    Image:         skydive/skydive:0.24.0
    Image ID:      docker.io/skydive/skydive@sha256:e2037c83f8cc0a45eb0947fe81e39e7e7ee46a920098c5fb6dbf0690c139c37b
    Ports:         8082/TCP, 8082/UDP, 12379/TCP
    Host Ports:    8082/TCP, 8082/UDP, 12379/TCP
    Args:
      analyzer
      --listen=0.0.0.0:8082
    State:          Running
      Started:      Sun, 09 Feb 2020 09:26:58 +0000
    Ready:          True
    Restart Count:  0
    Limits:
      cpu:     2
      memory:  8Gi
    Requests:
      cpu:      100m
      memory:   512Mi
    Liveness:   http-get http://:8082/api/status delay=20s timeout=1s period=10s #success=1 #failure=10
    Readiness:  http-get http://:8082/api/status delay=10s timeout=1s period=10s #success=1 #failure=3
    Environment:
      SKYDIVE_UI:                                {"theme":"light","k8s_enabled":"true"}
      SKYDIVE_ANALYZER_TOPOLOGY_PROBES:          k8s
      SKYDIVE_EMBEDDED:                          true
      SKYDIVE_FLOW_PROTOCOL:                     websocket
      SKYDIVE_ANALYZER_TOPOLOGY_FABRIC:          TOR1->*[Type=host]/eth0
      SKYDIVE_LOGGING_LEVEL:                     INFO
      SKYDIVE_FLOW_UPDATE:                       30
      SKYDIVE_ANALYZER_STARTUP_CAPTURE_GREMLIN:  G.V().has('Name', NE('lo'))
      SKYDIVE_ANALYZER_STARTUP_CAPTURE_BPF:      not (tcp dst port 8082)
      SKYDIVE_FLOW_DEFAULT_LAYER_KEY_MODE:       L3
      SKYDIVE_ANALYZER_STARTUP_CAPTURE_TYPE:     pcap
      SKYDIVE_AGENT_CAPTURE_SYN:                 True
    Mounts:
      /etc/ssl/certs from ssl (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from skydive-service-account-token-vzqsg (ro)
  skydive-exporter:
    Container ID:   containerd://5e57ff171ffbefb578401ec1025a6ce69812f3bc9425ef25a7f701d9b1852c71
    Image:          quay.io/sacloud/skydive-exporter:4
    Image ID:       quay.io/sacloud/skydive-exporter@sha256:22ab8d185a7da906b316b64e975d9e0fcb3ef620c1b9fce58196de25e0d7ea90
    Port:           <none>
    Host Port:      <none>
    State:          Running
      Started:      Sun, 09 Feb 2020 09:27:14 +0000
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Sun, 09 Feb 2020 09:26:58 +0000
      Finished:     Sun, 09 Feb 2020 09:27:13 +0000
    Ready:          True
    Restart Count:  1
    Environment:
      SKYDIVE_ANALYZERS:                                       skydive-skydive-service:8082
      SKYDIVE_PIPELINE_SUBSCRIBER_URL:                         ws://skydive-skydive-service:8082/ws/subscriber/flow
      SKYDIVE_PIPELINE_STORE_BUFFERED_FILENAME_PREFIX:         <set to the key 'objectPrefix' of config map 'skydive-exporter-s3-configuration'>  Optional: false
      SKYDIVE_PIPELINE_WRITE_S3_ENDPOINT:                      <set to the key 'endpoint' of config map 'skydive-exporter-s3-configuration'>      Optional: false
      SKYDIVE_PIPELINE_WRITE_S3_ACCESS_KEY:                    <set to the key 'accesskey' in secret 'skydive-exporter-secret'>                   Optional: false
      SKYDIVE_PIPELINE_WRITE_S3_SECRET_KEY:                    <set to the key 'secretkey' in secret 'skydive-exporter-secret'>                   Optional: false
      SKYDIVE_PIPELINE_STORE_BUFFERED_DIRNAME:                 <set to the key 'bucket' of config map 'skydive-exporter-s3-configuration'>        Optional: false
      SKYDIVE_PIPELINE_WRITE_S3_REGION:                        <set to the key 'region' of config map 'skydive-exporter-s3-configuration'>        Optional: false
      SKYDIVE_PIPELINE_STORE_BUFFERED_MAX_FLOWS_PER_OBJECT:    60000
      SKYDIVE_PIPELINE_STORE_BUFFERED_MAX_SECONDS_PER_OBJECT:  60
      SKYDIVE_PIPELINE_STORE_BUFFERED_MAX_FLOW_ARRAY_SIZE:     100000
      SKYDIVE_PIPELINE_STORE_BUFFERED_MAX_SECONDS_PER_STREAM:  86400
      SKYDIVE_PIPELINE_CLASSIFY_TYPE:                          subnet_autodiscovery
      SKYDIVE_PIPELINE_CLASSIFY_CLUSTER_NET_MASKS:             10.0.0.0/8 172.16.0.0/12 192.168.0.0/16
      SKYDIVE_PIPELINE_MANGLE_TYPE:                            none
      SKYDIVE_PIPELINE_FILTER_EXCLUDED_TAGS:                   other
      SKYDIVE_PIPELINE_ENCODE_TYPE:                            secadvisor
      SKYDIVE_PIPELINE_COMPRESS_TYPE:                          gzip
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from skydive-service-account-token-vzqsg (ro)
  skydive-minio:
    Container ID:   containerd://10a225f5b9ec2395764a7fe750cf671e82c047d60061e4db3494cfcb02e1d7fa
    Image:          docker.io/bitnami/minio:2019.7.31-debian-9-r1
    Image ID:       docker.io/bitnami/minio@sha256:f4ab6fa8c7ce912a1b67f57b14afb41bbf97d63e7820db83c8b8b5b15b8d0f67
    Port:           9000/TCP
    Host Port:      9000/TCP
    State:          Running
      Started:      Sun, 09 Feb 2020 09:26:58 +0000
    Ready:          True
    Restart Count:  0
    Environment:
      MINIO_ACCESS_KEY:       admin
      MINIO_SECRET_KEY:       admin1234
      MINIO_DEFAULT_BUCKETS:  default
      MINIO_REGION_NAME:      default
    Mounts:
      /data from data (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from skydive-service-account-token-vzqsg (ro)
Conditions:
  Type              Status
  Initialized       True
  Ready             True
  ContainersReady   True
  PodScheduled      True
Volumes:
  ssl:
    Type:          HostPath (bare host directory volume)
    Path:          /etc/ssl/certs
    HostPathType:
  data:
    Type:    EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
  skydive-service-account-token-vzqsg:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  skydive-service-account-token-vzqsg
    Optional:    false
QoS Class:       Burstable
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 600s
                 node.kubernetes.io/unreachable:NoExecute for 600s
Events:
  Type    Reason     Age                 From                   Message
  ----    ------     ----                ----                   -------
  Normal  Scheduled  103s                default-scheduler      Successfully assigned default/skydive-skydive-analyzer-6cc575468f-sjg77 to 10.74.144.77
  Normal  Pulled     103s                kubelet, 10.74.144.77  Container image "skydive/skydive:0.24.0" already present on machine
  Normal  Created    103s                kubelet, 10.74.144.77  Created container
  Normal  Started    102s                kubelet, 10.74.144.77  Started container
  Normal  Pulled     102s                kubelet, 10.74.144.77  Container image "docker.io/bitnami/minio:2019.7.31-debian-9-r1" already present on machine
  Normal  Created    102s                kubelet, 10.74.144.77  Created container
  Normal  Started    102s                kubelet, 10.74.144.77  Started container
  Normal  Pulled     87s (x2 over 102s)  kubelet, 10.74.144.77  Container image "quay.io/sacloud/skydive-exporter:4" already present on machine
  Normal  Created    87s (x2 over 102s)  kubelet, 10.74.144.77  Created container
  Normal  Started    86s (x2 over 102s)  kubelet, 10.74.144.77  Started container


```

**Pull request merged**